packages/react-native-sdk/src/components/call/CallControls.tsx

```diff
@@ -54,29 +61,50 @@ export const CallControls = ({
   hangUpCallButton,
   style,
 }: CallControlsType) => {
-  const [isReactionModalActive, setIsReactionModalActive] =
-    useState<boolean>(false);
+  const [showReactionsPopup, setShowReactionsPopup] = useState<boolean>(false);
+
+  const onOpenReactionsPopupHandler = useCallback(() => {
+    setShowReactionsPopup(true);
+  }, []);
+
+  const onCloseReactionsPopupHandler = useCallback(() => {
+    setShowReactionsPopup(false);
+  }, []);
```

Comment:
Instead of popup, I think `picker` would be a better name here

```suggestion
  const openReactionsPicker = useCallback(() => {
    setShowReactionsPopup(true);
  }, []);

  const closeReactionsPicker = useCallback(() => {
    setShowReactionsPopup(false);
  }, []);
```